### ISSUE TITLE
Can support the Additional Controller Advertising Data(ACAD) feature when using the BLE5.0 extended advertising 

### ISSUE THREAD

**Issue by xiaolongba** [espressif/esp-idf#10172]
### Answers checklist.

- [X] I have read the documentation [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/) and the issue is not addressed there.
- [X] I have updated my IDF branch (master or release) to the latest version and checked that the issue is present there.
- [X] I have searched the issue tracker for a similar issue and not found a similar issue.

### General issue report

Hello Experts,

Recently, I was debugging the BLE v5.0 extended advertising. based on the description of **BLUETOOTH SPECIFICATION Version 5.0 | Vol 6, Part B | 2.3.4.8  ACAD field**, Extended advertising can carry the (ACAD), but I cannot find the related APIs that can configure the data of this ACAD field. so in my understanding, the current esp-idf cannot support this feature yet? or any APIs I missed?

Concerning this issue, could you please give me a hand?

thanks in advance.

**Comment by xiewenxiang** [comment]
This field is Additional Controller Advertising Data field, there is no standard HCI command to set this field.
If must to set this field, it maybe should use vendor HCI commands.
This is not supported for now.

**Comment by xiaolongba** [comment]
Hello @xiewenxiang Bro,

I agree with you, but the Additional Controller Advertising Data field is not out of Bluetooth spec although there is not corresponding HCI command that can configure it.

so it is possible to implement this feature next release? or any workarounds can be used for this purpose?

thanks a lot in advance.

**Comment by xiewenxiang** [comment]
Hi @xiaolongba 
In Core_spec 5.2, this field doesn't really have any specific use, there are more detailed instructions in 5.2

Such as:

`BLUETOOTH CORE SPECIFICATION Version 5.2 | Vol 3, Part C.  Page 1371`

`A device in the Broadcast Isochronous Synchronizability mode shall send the BIGInfo in the ACAD field which is located in the AUX_SYNC_IND PDU of periodic advertisement `

The controller populates this field automatically when it populates BIGinfo, Therefore, my understanding is that this field should not be used to actively configure the host. I am not sure if my understanding is correct....

<img width="844" alt="image" src="https://user-images.githubusercontent.com/26754847/201808222-b60934a7-415f-4ca0-ae5f-abeb28f35d68.png">

**Comment by xiaolongba** [comment]
Hello @xiewenxiang Bro,

From Bluetooth v5.0 spec, it doesn't support the BIS info which is used for LE audio, but the ACAD field is still present, so in my opinion, BLE v5.0 shall support this even using the host. Could you help me double-check?

Thanks in advance.

**Comment by xiewenxiang** [comment]
@xiaolongba  We'll evaluate it and get back to you.

**Comment by xiewenxiang** [comment]
Hi @xiaolongba 
We talked about it, and we thought although no specific usage was given in 5.0, subsequent releases have been given specific usage scenarios, so, this field is just for controller.

We also want to know why you want to set this field, The Host Advertising filed should be have enough length for the user to use.

**Comment by xiaolongba** [comment]
Hello @xiewenxiang bro,

Indeed, it is only for the controller, is it support this feature in terms of the controller?

The purpose why I asked this question is that I am an educator who is writing some tutorials based on esp32-c3, the link is below:
https://github.com/xiaolongba/HX_DK_FOR_ESP32-C3/wiki 

At this moment, I was trying to explain the new features of the esp32-c3.

So is it possible to give me a hand?

Thanks a lot in advance.

**Comment by xiewenxiang** [comment]
Hi @xiaolongba,

You wrote a great tutorial! I'm sure many people will use your tutorial to learn more detail about ESP32-C3.

For the ACAD field, I think you can refer directly to the spec description. It will be auto configured by controller.

**Comment by xiaolongba** [comment]
@xiewenxiang Yes, but I can not find which APIs can trigger this operation. Could you give me some tips?
it seems that spec  has not disclosed too much Info about it,Only introduction for the functionality.

**Comment by xiewenxiang** [comment]
Yes, the spec does not provide an interface for setting this field, so I think you can explain in the tutorial that this field cannot be set manually.

**Comment by xiaolongba** [comment]
> Yes, the spec does not provide an interface for setting this field, so I think you can explain in the tutorial that this field cannot be set manually.

ok, the last question, the current controller of esp32-c3 can support this feature?

**Comment by xiewenxiang** [comment]
The controller of C3 currently has code to set this field, but don't support Host to set it.

**Comment by xiaolongba** [comment]
> The controller of C3 currently has code to set this field, but don't support Host to set it.

Understood, you can close this ticket.

Thanks.

**Comment by Alvin1Zhang** [comment]
Thanks for sharing, feel free to reopen.
